src/test/java/org/cactoos/text/ComparableTextTest.java

```diff
@@ -68,4 +71,52 @@ public void comparesToUncheckedText() {
             Matchers.equalTo(0)
         );
     }
+
+    @Test
+    public void equalsAndHashCodeOfComparableOfTheSameText() {
+        final Text text = new TextOf("my text");
+        final Text actual = new ComparableText(text);
+        final Text expected = new ComparableText(text);
+        MatcherAssert.assertThat(
```

Comment:
@ilyakharlamov Use `Assertion` instead.

Reply:
@fabriciofx  Assertion is in cactoos-matchers since 0.13, it's not available yet

Reply:
@ilyakharlamov Let's wait for #1023.

Reply:
@Iprogrammerr #1023 has solved. Please, go ahead.

Reply:
done